jdaviz/configs/default/plugins/markers/markers.py

```diff
@@ -107,15 +113,60 @@ def clear_table_callback():
         self.hub.subscribe(self, RemoveDataMessage,
                            handler=lambda msg: self._recompute_mark_positions(msg.viewer))
 
-        self.docs_description = "Press 'm' with the cursor over a viewer to log\
+        self.docs_description = """Press 'm' with the cursor over a viewer to log\
                                 the mouseover information. To change the\
                                 selected layer, click the layer cycler in the\
                                 mouseover information section of the app-level\
-                                toolbar."
+                                toolbar.
+Press 'd' twice to measure the distance between any two points. Hold the 'Option'
+(Mac) or 'Alt' key while pressing 'd' to snap measurement points to the nearest
+marker."""
 
         # description displayed under plugin title in tray
         self._plugin_description = 'Create markers on viewers.'
 
+    @traitlets.default('distances_table')
+    def _default_distances_table(self):

```

Comment:
interesting!  We don't use this anywhere else but I see how it can be useful here - maybe just leave a note where the traitlet is instantiated that the defaults are set here